packages/adapter-d1/src/d1.ts

```diff
@@ -176,6 +176,8 @@ export class PrismaD1 extends D1Queryable<StdClient> implements DriverAdapter {
     const tag = '[js::startTransaction]'
     // console.debug(`${tag} options: %O`, options)
 
+    console.warn('D1 does not currently provide transactions support, as such, we cannot guarantee atomicity')
```

Comment:
```suggestion
    console.warn('Cloudflare D1, currently in Beta, does not support transactions. When using Prisma's D1 adapter, implicit & explicit transactions will be ignored and ran as individual queries, which breaks the guarantees of the ACID properties of transactions. For more details see https://pris.ly/d/d1-transactions')
```
As we don't have any D1 docs at the moment, we could point that to a discussion like a feedback discussion, for now. And there we can mention https://github.com/cloudflare/workers-sdk/issues/2733